servicecontrol/contracts.md

```diff
@@ -16,6 +16,8 @@ redirects:
 
 ServiceControl events allow to build notifications/integrations that will alert of something going wrong in the system.
 
+WARNING: ServiceControl sends external notification events in batches. If a problem is encountered part way through a batch, the entire batch will be resent. This can result in receiving mutliple events for a single notification. 
```

Comment:
Well technically they have a new message Id so they are different messages. Just dropping Outbox in won't fix it. For that reason I wanted to be very careful about this was phrased.